lib/plan-finder.js

```diff
@@ -152,22 +153,38 @@ class PlanFinder {
                 * @param {String} name transformer name
                 * @param {Object} attributes custom attributes for this step
      */
-    formPlan(source, outputInstructions, currentPlan) {
+    async formPlan(source, outputInstructions, currentPlan) {
```

Comment:
I tried out an e2e request in the devtool with a small enough file that has orientation. it didn't end up running exiftool or adding an extra step to the plan:
[logs](https://adobeiort.splunkcloud.com/en-US/app/search/search?q=search%20index%3D%22runtime_prod_ext%22%20activation_id%3D550d48b4c5b042188d48b4c5b0521807%20%7C%20sort%20_time%20%7C%20fields%20%2B%20_time%2Clog_message%20%7C%20fields%20-%20_raw%20%7C%20outputtext&display.page.search.mode=smart&dispatch.sample_ratio=1&earliest=%40d&latest=now&display.page.search.showFields=0&display.events.type=table&display.page.search.tab=statistics&display.general.type=statistics&sid=1629243000.631671)
activationId: `550d48b4c5b042188d48b4c5b0521807`

I suggest adding some logs in this method to see whats happening:
```
console.log( `current plan: ${currentPlan}`
```

Reply:
fyi, the file in azure blob meahana is: landscape_3_319_.jpg